src/handwritten/task.py

```diff
@@ -402,6 +402,52 @@ def is_task_done(self):
 
         return is_task_done
 
+    def perform_bridge_offset_nulling_cal(self, channel="", skip_unsupported_channels=False):
+        """
+        Perform a bridge offset nulling calibration on the channels in the task.
+
+        If the task measures both bridge-based sensors and non-bridge-based sensors,
+        use the channels input to specify the names of the channels that measure
+        bridge-based sensors.
+
+        Args:
+            channel: is a subset of virtual channels in the task that you want to calibrate.
+            Use this input if you do not want to calibrate all the channels in the task or
+            if some channels in the task have open thermocouple detection disabled.
+            If the input is empty, this VI attempts to calibrate all virtual channels in the task.
+
+            skip_unsupported_channels: specifies whether or not to skip channels that do not 
+            support calibration.
+            If skip unsupported channels is TRUE, this VI calibrates only supported channels.
+            If FALSE, this VI calibrates the channels specified by channels. The default is FALSE.
```

Comment:
If a parameter has more than one line, you need to indent the rest. See https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html for examples.

This should display 2 parameters, but it's interpreting parts of the other lines as parameters.
![image](https://github.com/ni/nidaqmx-python/assets/17914399/e1883fac-02bd-4d2f-9112-506a0e57c7f0)

Reply:
Thanks for pointing that out, I have indented it and the result is as shown.
![image](https://github.com/ni/nidaqmx-python/assets/74756143/14a789e3-d981-4ad0-a832-0524a3d9ea45)